examples/native/components/BackgroundExample/BackgroundCsf.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react-native';
+import { withBackgrounds } from '@storybook/addon-ondevice-backgrounds';
+import { Dimensions, Text, View, StyleSheet } from 'react-native';
+
+const Background = () => (
+  <View style={styles.view}>
+    <Text style={styles.text}>Change background color via Addons -&gt; Background</Text>
+  </View>
+);
+
+const styles = StyleSheet.create({
+  view: { height: Dimensions.get('window').height },
```

Comment:
I don't know why this is necessary: For some reason, the text refused to appear until I added this View with an explicit definition of the height. This hack is also included in the example in `BackgroundPanel.tsx` (which is a copy-paste of this file).

Didn't see the same phenomenon with the other CSF stories. Also the `storiesOf` example just renders the `<Text>` component without any wrappers and it seems to work fine 🤔

Reply:
So I found out what the problem is, the decorator in preview.js with the padding has no height. If you give it the flex:1 style then the text here will work as normal.

Reply:
I committed the fix to your branch :)

Reply:
Awesome, thanks for fixing it and for explaining what was happening! 👍  🙏